.prettierrc.yml

```diff
@@ -0,0 +1,7 @@
+semi: true
+trailingComma: 'all'
+singleQuote: true
+printWidth: 80
```

Comment:
I don't love this.  You've got yml/comments/json all going more like 100 characters, and wrapping some of these js lines at 80 makes them look funky.

Reply:
Prettier is even more opinionated than usual about [that](https://prettier.io/docs/en/options.html#print-width), so I went with the default.

Reply:
Ugh... yeah, someone on that project really wants to die on that hill.  I'm going to make my own linter that argues for 40 character lines.